deploy/nexodus/base/apiserver/deployment.yaml

```diff
@@ -13,88 +13,88 @@ spec:
           image: quay.io/nexodus/apiserver:latest
           imagePullPolicy: IfNotPresent
           env:
-            - name: APEX_DEBUG
+            - name: DEBUG
```

Comment:
All of these should be fixed based on my suggestion to re-add the project name prefix